R/post_bug.R

```diff
@@ -8,8 +8,8 @@ components <- c("Accuracy", "Add-ons", "Analyses", "Documentation", "Graphics",
 #'
 #' Guides through the process of creating an issue.
 #' Requires an user and an API key.
-#' @param text A character vector with the text of the bug you want to
-#' open.
+# #' @param text A character vector with the text of the bug you want to
+# #' open.
```

Comment:
Remember that text which needs to be the description of the issue needs to be included. It is the most important part of posting an issue! Even if the API allows to post without a description